### DISCUSSION TITLE
Community servers

### DISCUSSION THREAD

**Discussion by hubertwas** [koka-lang/koka#504] [General]
Are there any community servers on Discord or Zulip or anything else?

**Comment by TimWhiting** [comment]
Not really anything currently except GitHub Discussions. There is a group of us collaborating on some shared Koka community libraries here: https://github.com/koka-community, with GitHub Discussions as well. Since there is enough people that have asked about using another platform, I have created a discord server associated with that group. https://discord.gg/CEkHHnTqEv

**Reply by chtenb** [reply]
I checked it out, but I noticed I couldn't join unless I put in a phone number. Is that a conscious requirement? This wasn't necessary for several other programming related servers I'm in, and I'm generally hesitant to share personal information on too many platforms.

**Reply by TimWhiting** [reply]
Sorry about that, I've changed the requirement to the level just below that. 

It is under the spam protection setup for a community server, and so it is mainly intended to discourage spammy accounts / bots. 

It didn't look like you would have to share the phone number, just that there was a verified phone number associated with your account so that we know that it is associated with a real person, but I didn't think about the norms of programming servers, or the fact that people might not have a phone connected for 2FA or for not wanting to store numbers in discord period. So it was a conscious requirement, but I wasn't set on it or completely sure it was the right choice. Thanks for recommending we don't require that.
